tests/.eslintrc.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   env: {
     'embertest': true
+  },
+  globals: {
+    'Factory': false,
```

Comment:
That doesn't scale very well and this is specifically in the test `.eslintrc`. Do you feel it's better that way?